scripts/validate_container.sh

```diff
@@ -131,7 +131,7 @@ do
     echo "Source: $ex"
     let "samples+=1"
 
-    if [[ "$ex" == *"iqm"* ]] || [[ "$ex" == *"ionq"* ]] || [[ "$ex" == *"quantinuum"* ]] || [[ "$ex" == *"nvcf"* ]];
+    if [[ "$ex" == *"iqm"* ]] || [[ "$ex" == *"ionq"* ]] || [[ "$ex" == *"quantinuum"* ]] || [[ "$ex" == *"nvqc"* ]];
```

Comment:
We should think about reenabling pre-merge checks for the nvqc backend - let's chat about this offline.

Reply:
Discussed offline. We'll enable pre-merge checks for NVQC once we have a semi-permanent/stable deployment.